Workade/Views&ViewModels/Home/HomeViewModel.swift

```diff
@@ -11,22 +11,28 @@ import UIKit
 final class HomeViewModel {
     var bookmarkManager = BookmarkManager.shared
     
-    var officeResource = OfficeResource(context: [])
+    var officeResource = OfficeResource(content: [])
```

Comment:
[P2] 현재 이니셜라이저의 형태에 다른 특별한 이유가 없다면 따로 `OfficeResource()`로 선언할 수 있게 인수를 받지 않는 이니셜라이저를 만들어 주는 것도 괜찮을 것 같습니다!!

Reply:
[P2] 모델 안에서 초기 배열을 넣어주는 형태로 수정하겠습니다~! 굳굳 좋은 포인트네용
```suggestion
struct OfficeResource: Codable {
    let content: [Office] = []
}
```

Reply:
아! 아니군요. 프로퍼티 자체에 [] 초깃값을 주려면 let 상수가 아닌 var를 써야하네요!
에반이 제시한 것처럼 init문을 따로 만들어서 처음 선언때는 매개변수를 쓰지않아도 되게하고, 추후 Decoding되어 들어오는 Resource는 아예 새로운 (멤버와이즈 이니셜라이저로 만들어진) 구조체가 officeResource 변수에 할당되는 느낌이 딱 좋겠네용
```suggestion
    struct OfficeResource: Codable {
    let content: [Office]
    
    init() {
        self.content = []
    }
```

Reply:
수정했습니다~!